pdns/ixfrdist.cc

```diff
@@ -619,8 +620,7 @@ void handleTCPRequest(int fd, boost::any&) {
     return;
   }
 
-  char buf[4096];
-  // Discard the first 2 bytes (qlen)
+  char buf[65535];
   int res;
```

Comment:
`recv()` returns a `ssize_t`, even if we should be fine given that the buffer is currently smaller than `2^16`